scripts/core/helpers/typescript-helpers.ts

```diff
@@ -1,3 +1,7 @@
 export function nameof<T>(name: keyof T) {
     return name;
 }
+
+export function assertNever(x: never): never {
```

Comment:
what is this function for?

Reply:
It makes exhaustive switch possible.

```
type sectionToRender = 'header' | 'content';

function handleIt (section: sectionToRender) {
    switch(section) {
        case 'header':
            // ...
            break;
        case 'content':
            // ...
            break;
        default:
            assertNever(section)
    }
}
```

If you add another possible value to `sectionToRender` type, the code won't compile until you handle it in all switch statements which switch on that type.

Reply:
Cool